ruby/README.md

```diff
@@ -56,6 +56,41 @@ inherit_from: rails_rubocop.yml
 # project-specific configuration...
 ```
 
+## Reek
+
+Every project should start with the `reek.yml` present here. It should
+enforce the styles defined here.
+
+It should be pulled in via [external configuration].
+
+```
+prepare:
+  fetch:
+  - url: "https://raw.githubusercontent.com/codeclimate/styleguide/master/ruby/reek.yml"
+    path: ".reek.yml"
+
+engines:
+  reek:
+    enabled: true
+```
+
+Reek does not support config inheritance, so you will not add a
+project-specific file.
+
+Reek has strong opinions about object-orientation that many find
+overaggressive in some parts of codebases where OO design is less
+important. You may find these common exclusions to be helpful:
```

Comment:
This was my main thought when I saw this PR. What you wrote here hits home for me. I think of `reek` as valuable, specially for growing teams. It helps drive some level of quality for some standards without too much PR micromanagement.

In some aspects, though, it's like a very purist tool. In some cases we need to make an executive decision to sacrifice something for a benefit somewhere else, and in those moments I found `reek` to be annoying. 

I'm interested in the list of rules you set up, it looks like you gave it some thought and also recognize that it's aggressive so I'm quite interested 😄